test/image/mocks/gl2d_parcoords_dark_background.json

```diff
@@ -0,0 +1,25 @@
+{
+  "layout": {
+    "width": 284,
+    "height": 400,
+    "font": {"color": "white"},
+    "paper_bgcolor": "black"
```

Comment:
Non-blocking:
Alternatively we could change the `paper_bgcolor` of one of our existing `parcoords` mocks.

Reply:
Nevermind. We may get some merge conflicts with the other PR. So feel free to keep it as is.

Reply:
After doing some testing with other mocks, I think it would be best to duplicate the data and layout from `gl2d_parcoords.json` mock here and then add
```js
    "font": {"color": "white"},
    "paper_bgcolor": "black"
```
That would really show (e.g. on the second axis) why the text shadow is needed i.e. to avoid mixing with background lines.

Reply:
I don't have the code pulled up right now, but does that mock have a constraint?  I wanted make sure there's a constraint bar to test the change to the left/right border color there as well.

Reply:
It does have constraints.

Reply:
Done in https://github.com/plotly/plotly.js/pull/5506/commits/a480991732aef7bbf83f76d0b5e8e28c8a08ebc9.

Thanks! This is much better